os-depends

```diff
@@ -63,6 +63,7 @@ openssh-client
 ostree
 p7zip-full
 parted
+pesign [amd64]
```

Comment:
I guess this tool is small, and I understand the desire to be able to check at the point the bootloader is updated that it's correctly signed, but – I think this check belongs in the ostree (and image) builder. We should never get as far as publishing unsigned bootloaders to end users.